src/helpers/constants/raw.queries.js

```diff
@@ -0,0 +1,9 @@
+const ratingsWithUsers = slug => `SELECT rating, 
+ "Users"."username", "Users"."email", "Users"."firstName", "Users"."lastName", "Users"."profileImage"
```

Comment:
@nkpremices Cool, I like the use of inner join here. I'm curious why you've used SQL and not Sequelize?

Reply:
Oh!!! 
This was actually my first solution. 
Now, after deleting that helper I've been using to compute data and do the pagination, I forgot to remove this file. 
I'm gonna remove it @MemunaHaruna